### ISSUE TITLE
[tox4] rendering documentation is broken

### ISSUE THREAD

**Issue by jugmac00** [tox-dev/tox#2321]
```
$ tox4 -e docs
.pkg: get_requires_for_build_wheel> python /home/jugmac00/opt/tox4/lib/python3.8/site-packages/pyproject_api/_backend.py True setuptools.build_meta
.pkg: build_wheel> python /home/jugmac00/opt/tox4/lib/python3.8/site-packages/pyproject_api/_backend.py True setuptools.build_meta
docs: install_package> python -I -m pip install --force-reinstall --no-deps /home/jugmac00/Projects/tox/.tox/4/.pkg/dist/tox-4.0.0a11.dev15+g4d788229-py3-none-any.whl
docs: commands[0]> sphinx-build -d /home/jugmac00/Projects/tox/.tox/4/docs/tmp/doctree docs /home/jugmac00/Projects/tox/.tox/4/docs_out --color -b html -W
Running Sphinx v4.3.2
Loading template...
Finding news fragments...
Rendering news fragments...
Draft only -- nothing has been written.
What is seen below is what would be written.

loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://packaging.pypa.io/en/latest/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 10 source files that are out of date
updating environment: [new config] 10 added, 0 changed, 0 removed
reading sources... [100%] user_guide                                                                                                                                
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] user_guide                                                                                                                                 

Warning, treated as error:
/home/jugmac00/Projects/tox/.tox/4/docs/lib/python3.8/site-packages/tox/config/cli/parser.py:docstring of tox.config.cli.parser.ArgumentParserWithEnvAndConfig.parse_args::py:class reference target not found: Namespace
docs: exit 2 (7.26 seconds) /home/jugmac00/Projects/tox> sphinx-build -d /home/jugmac00/Projects/tox/.tox/4/docs/tmp/doctree docs /home/jugmac00/Projects/tox/.tox/4/docs_out --color -b html -W pid=1556436
.pkg: _exit> python /home/jugmac00/opt/tox4/lib/python3.8/site-packages/pyproject_api/_backend.py True setuptools.build_meta
  docs: FAIL code 2 (10.64=setup[3.38]+cmd[7.26] seconds)
  evaluation failed :( (10.86 seconds)
```

Not sure how this is possible as we test doc building also on CI ... oO

## update

Locally, tox renders the doc via Python 3.8 - on gha Python 3.10 is set.

When I set `basepython = python3.10` in tox.ini, rendering docs works for me, too.

Is this related to `from __future__ import annotations`?

Can we add `basepython=python3.10` to the docs env?

**Comment by gaborbernat** [comment]
Sure. I'm working on the assumption that by default everyone is using it nowadays. Python 3.10.  I mean why would they not do so?  If you can, you should use the latest Python by default.  So just install tox  under Python, 3.10 and everything is good.

**Comment by jugmac00** [comment]
While I possibly could, I think we cannot expect from all contributors to have tox installed on Python 3.10 or higher. I'll create a PR to add a basepython for the docs env.

**Comment by jugmac00** [comment]
Fixed by #2322 (not sure why my comment on the PR did not close this issue)

**Comment by gaborbernat** [comment]
> Fixed by #2322 (not sure why my comment on the PR did not close this issue)

The auto close only works when you merge into the default branch (currently master v3).
